tools/svsm/Launch_SVSM.sh

```diff
@@ -0,0 +1,596 @@
+#!/bin/bash
+#  SPDX-License-Identifier: MIT
+#
+#  Copyright (C) 2023 Advanced Micro Devices, Inc.  
+
+
+set -eE
+set -o pipefail
+[ "$DEBUG" == 'true' ] && set -x
+
+#trap cleanup EXIT
+
+# Working directory setup
+WORKING_DIR="${WORKING_DIR:-$HOME/coconut-svsm}"
+SETUP_WORKING_DIR="${SETUP_WORKING_DIR:-${WORKING_DIR}/setup}"
+LAUNCH_WORKING_DIR="${LAUNCH_WORKING_DIR:-${WORKING_DIR}/launch}"
+ATTESTATION_WORKING_DIR="${ATTESTATION_WORKING_DIR:-${WORKING_DIR}/attest}"
+
+# Export environment variables
+COMMAND="help"
+SKIP_IMAGE_CREATE=false
+HOST_SSH_PORT="${HOST_SSH_PORT:-10022}"
+GUEST_NAME="${GUEST_NAME:-snp-guest}"
+GUEST_SIZE_GB="${GUEST_SIZE_GB:-50}"
+GUEST_MEM_SIZE_MB="${GUEST_MEM_SIZE_MB:-2048}"
+GUEST_SMP="${GUEST_SMP:-4}"
+CPU_MODEL="${CPU_MODEL:-EPYC-v4}"
+GUEST_USER="${GUEST_USER:-amd}"
+GUEST_PASS="${GUEST_PASS:-amd}"
+GUEST_SSH_KEY_PATH="${GUEST_SSH_KEY_PATH:-${LAUNCH_WORKING_DIR}/${GUEST_NAME}-key}"
+GUEST_ROOT_LABEL="${GUEST_ROOT_LABEL:-cloudimg-rootfs}"
+GUEST_KERNEL_APPEND="root=LABEL=${GUEST_ROOT_LABEL} ro console=ttyS0"
+QEMU_CMDLINE_FILE="${QEMU_CMDLINE:-${LAUNCH_WORKING_DIR}/qemu.cmdline}"
+IMAGE="${IMAGE:-${LAUNCH_WORKING_DIR}/${GUEST_NAME}.img}"
+GENERATED_INITRD_BIN="${SETUP_WORKING_DIR}/initrd.img"
+
+# URLs and repos
+CLOUD_INIT_IMAGE_URL="https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img"
+
+usage() {
+  >&2 echo "Usage: $0 [OPTIONS] [COMMAND]"
+  >&2 echo "  where COMMAND must be one of the following:"
+  >&2 echo "    setup-host            Build required SNP components and set up host"
+  >&2 echo "    launch-guest          Launch a SNP guest"
+  >&2 echo "    attest-guest          Use virtee/snpguest and sev-snp-measure to attest a SNP guest"
+  >&2 echo "    stop-guests           Stop all SNP guests started by this script"  
```

Comment:
Is there a particular reason we are outputting usage to stderr instead of stdio?

Reply:
Directing usage information or errors to stderr helps separate errors from normal output, making it easier to handle in automated environments. Many tools monitor stderr to detect issues that need attention.